src/System.Management.Automation/engine/NativeCommandProcessor.cs

```diff
@@ -1108,14 +1125,14 @@ private void ProcessOutputRecord(ProcessOutputObject outputValue)
         }
 
         /// <summary>
-        /// Gets the start info for process.
+        /// Gets the ProcessStartInfo for process.
         /// </summary>
-        /// <param name="redirectOutput"></param>
-        /// <param name="redirectError"></param>
-        /// <param name="redirectInput"></param>
-        /// <param name="soloCommand"></param>
-        /// <returns></returns>
-        private ProcessStartInfo GetProcessStartInfo(bool redirectOutput, bool redirectError, bool redirectInput, bool soloCommand)
+        /// <param name="redirectOutput">A boolean which determines whether the output is to be redirected.</param>
+        /// <param name="redirectError">A boolean which determines whether errors are to be redirected.</param>
+        /// <param name="redirectInput">A boolean which determines whether the input is to be redirected.</param>
+        /// <param name="soloCommand">A boolean which determines whether this is a solo command.</param>
```

Comment:
This comment just restates what the variable name already says, but I don't know enough to provide an alternative here. Please change the comment to supplement the information in the variable name.